module/Application/config/module.config.php

```diff
@@ -1,9 +1,13 @@
 <?php
 /**
  * @license   http://opensource.org/licenses/BSD-3-Clause BSD-3-Clause
- * @copyright Copyright (c) 2014 Zend Technologies USA Inc. (http://www.zend.com)
+ * @copyright Copyright (c) 2014-2016 Zend Technologies USA Inc. (http://www.zend.com)
  */
 
+namespace Application;
+
+use Zend\ServiceManager\Factory\InvokableFactory;
+
 return array(
```

Comment:
:blush: We never setup phpcs for this. I'll add it now, and have it scan for issues.